packages/libs/hd-wallet/README.md

```diff
@@ -51,128 +51,263 @@ The library is divided into two parts:
 
 Below are some common use cases for the `@kadena/hd-wallet` library.
 
-### Generating a Mnemonic
+## Key Concepts
 
-A mnemonic is a set of words that can be used to generate a seed. You can
-generate a mnemonic using `kadenaGenMnemonic`:
+- **Mnemonic Phrase**: A human-readable set of words used to generate a seed.
+- **Seed**: A binary representation of the mnemonic, which can be used to derive
```

Comment:
this is not correct! *Entropy* is the representation of the mnemonic not seed. 
Seed is a binary derived from mnemonic (with a hashing algorithm) will be used for creating master private key. Basically you can't generate mnemonic from a seed but you can create it from Entropy.